diff --git a/node_modules/expo-dev-launcher/android/build.gradle b/node_modules/expo-dev-launcher/android/build.gradle
index 6986ac5..ffcce29 100644
--- a/node_modules/expo-dev-launcher/android/build.gradle
+++ b/node_modules/expo-dev-launcher/android/build.gradle
@@ -88,7 +88,6 @@ repositories {
     url "$rootDir/../node_modules/jsc-android/dist"
   }
   google()
-  jcenter()
 }
 
 dependencies {
