diff --git a/node_modules/react-native-tesseract-ocr/android/build.gradle b/node_modules/react-native-tesseract-ocr/android/build.gradle
index 885c6e5..dd3554b 100644
--- a/node_modules/react-native-tesseract-ocr/android/build.gradle
+++ b/node_modules/react-native-tesseract-ocr/android/build.gradle
@@ -30,7 +30,6 @@ buildscript {
     if (project == rootProject) {
         repositories {
             google()
-            jcenter()
         }
         dependencies {
             classpath 'com.android.tools.build:gradle:3.4.1'
@@ -67,7 +66,6 @@ repositories {
         url "$rootDir/../node_modules/jsc-android/dist"
     }
     google()
-    jcenter()
 }
 
 dependencies {
diff --git a/node_modules/react-native-tesseract-ocr/android/src/main/java/com/reactlibrary/TesseractOcrModule.java b/node_modules/react-native-tesseract-ocr/android/src/main/java/com/reactlibrary/TesseractOcrModule.java
index e93e31c..72d6fa3 100644
--- a/node_modules/react-native-tesseract-ocr/android/src/main/java/com/reactlibrary/TesseractOcrModule.java
+++ b/node_modules/react-native-tesseract-ocr/android/src/main/java/com/reactlibrary/TesseractOcrModule.java
@@ -3,7 +3,6 @@ package com.reactlibrary;
 import android.graphics.Bitmap;
 import android.graphics.BitmapFactory;
 import android.graphics.Rect;
-import android.os.Environment;
 import android.util.Log;
 
 import androidx.annotation.Nullable;
@@ -34,16 +33,14 @@ public class TesseractOcrModule extends ReactContextBaseJavaModule {
     private static final String KEY_TOKEN_LEVEL = "level";
     private static final String TESS_FILES_DIRECTORY = "tessdata";
     private static final String TESS_FILES_EXTENSION = ".traineddata";
-    private static String DATA_PATH = Environment.getExternalStorageDirectory().toString();
+    private static String DATA_PATH;
     private static String TESS_FILES_PATH;
     private TessBaseAPI tesseract;
 
     public TesseractOcrModule(ReactApplicationContext reactContext) {
         super(reactContext);
         this.reactContext = reactContext;
-        if (!this.DATA_PATH.contains(reactContext.getPackageName())) {
-            this.DATA_PATH += File.separator + reactContext.getPackageName();
-        }
+        this.DATA_PATH = reactContext.getCacheDir().toString();
         this.TESS_FILES_PATH = this.DATA_PATH + File.separator + this.TESS_FILES_DIRECTORY;
     }
 
